test/versioned/cassandra-driver/package.json

```diff
@@ -8,7 +8,7 @@
         "node": ">=12"
       },
       "dependencies": {
-        "cassandra-driver": ">=3.4.0 <4"
+        "cassandra-driver": ">=3.4.0 <5"
```

Comment:
Do we need to cap at 5? Or can we leave that open-ended?

Reply:
We could certainly leave it open ended. I'll make the change.